### DISCUSSION TITLE
FYI: once minimum CMake version bumps to 3.25, JSON_SystemInclude is no longer needed

### DISCUSSION THREAD

**Discussion by starball5** [nlohmann/json#4418] [General]
with 3.25, anyone depending upon this library via `FetchContent` or `add_subdirectory` can use their `SYSTEM` arguments (see also https://stackoverflow.com/a/74311280/11107541).

**Comment by nlohmann** [comment]
Thanks for the heads-up! Good to keep on file for whenever the CMake minimum version is raised in the future — at that point, the workaround/interface option could potentially be simplified using CMake's native SYSTEM support for FetchContent/add_subdirectory.

---
*This reply was drafted by [Claude Code](https://claude.com/claude-code) on behalf of @nlohmann, as part of a review of open discussions.*
